macosx/FileOutlineController.mm

```diff
@@ -116,10 +110,10 @@ - (void)setFilterText:(NSString*)text
         {
             FileListNode* parent = nil;
             NSUInteger previousIndex = !item.isFolder ?
-                [self findFileNode:item inList:self.fFileList
-                         atIndexes:[NSIndexSet indexSetWithIndexesInRange:NSMakeRange(currentIndex, self.fFileList.count - currentIndex)]
-                     currentParent:nil
-                       finalParent:&parent] :
+                [self.class findFileNode:item inList:self.fFileList
```

Comment:
Do we really need to make this a class method?

Reply:
It's a personal habit to keep outside the instance implementation any piece of code that is independent from the instance.
But since it doesn't affect the fix here, I've now reverted that part and kept it as an instance method.